src/main/java/de/tum/in/www1/artemis/web/rest/CourseResource.java

```diff
@@ -1216,13 +1216,13 @@ public void fetchParticipationsWithSubmissionsAndResultsForCourses(List<Course>
     }
 
     /**
-     * Post /courses/:courseId/tutors/:tutorLogin : Add the given user to the tutors of the course so that the student can access the course administration
+     * POST courses/:courseId/tutors/:tutorLogin : Add the given user to the tutors of the course so that the student can access the course administration
```

Comment:
```suggestion
     * Post /courses/:courseId/tutors/:tutorLogin : Add the given user to the tutors of the course so that the student can access the course administration
```